sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -472,18 +472,19 @@ void gnrc_ipv6_nib_handle_timer_event(void *ctx, uint16_t type)
 void gnrc_ipv6_nib_change_rtr_adv_iface(gnrc_netif_t *netif, bool enable)
 {
     gnrc_netif_acquire(netif);
-    if (enable) {
+    if (enable && !gnrc_netif_is_rtr_adv(netif)) {
         _set_rtr_adv(netif);
     }
-    else {
-        uint32_t next_rs_time = random_uint32_range(0, NDP_MAX_RS_MS_DELAY);
-
+    else if (!enable && gnrc_netif_is_rtr_adv(netif)) {
```

Comment:
Question is whether we want to have this change, so that an already advertising interface does not trigger another RA here by the call to `gnrc_ipv6_nib_change_rtr_adv_iface`. I thought it would be a fix but previously I saw that an RA can be triggered from our `ifconfig` also when the router is already advertising.

Reply:
Even if this was not the original intention behind that behavior, I found it to be quite useful for debugging (manually triggering the sending of a RA) - so I'd say let's keep the original behavior (no checking of current RTR_ADV state)